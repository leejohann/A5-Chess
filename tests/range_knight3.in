move 6 7 5 5
quit
