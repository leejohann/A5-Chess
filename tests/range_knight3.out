RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _ _ 
pppppppp
rnbqkbnr

it is White's turn
RNBQKBNR
PPPPPPPP
 _ _ _ _
_ _ _ _ 
 _ _ _ _
_ _ _n_ 
pppppppp
rnbqkb_r

it is Black's turn

